Python/Product/Django/Django.csproj

```diff
@@ -113,6 +113,31 @@
     </Reference>
     <Reference Include="WindowsBase" />
   </ItemGroup>
+  <Choose>

```

Comment:
I would actually be surprised if they can handle references inside `Choose`, but not imports. If they get the `Reference` items from MSBuild, it should take care of it. If they parse unevaluated items themselves, they probably aren't handling conditions right, anyway (beyond the most basic stuff like configurations).